master.tex

```diff
@@ -63,14 +63,14 @@
 
 % This is the only place needed to change the current version of LedgerSMB
 % that this document references. Keep the space after.
-\newcommand{\ledgerSMBversion}{1.9 }
+\newcommand{\ledgerSMBversion}{1.10 }
 
 \begin{document}
 \include{acronyms}
 \include{glossary}
 
 
-\author{Erik H\"ulsmann, Chris Travers}
+\author{Erik H\"ulsmann, Chris Travers, Neil Tiffin}

```

Comment:
Yep. Maybe we need a contributors section too?

Reply:
Do you have other names that should be in the contributors section?

Reply:
Not at the moment. Judging by the commits, it's just you and me for now. But I hope that will change!

Reply:
Done.  I could not get the standard title page to work, so I converted to a custom title page. Let me know what you think. I am not wedded to the format.